tests/conftest.py

```diff
@@ -121,3 +121,20 @@ def disable_query_cache() -> Generator[None, None, None]:
     state.set_configs({"use_cache": 0, "use_readthrough_query_cache": 0})
     yield
     state.set_configs({"use_cache": cache, "use_readthrough_query_cache": readthrough})
+
+
+@pytest.fixture
+def set_state() -> Generator[Callable[[str, Any], None], None, None]:
```

Comment:
Is this required? Updating config should be a one-liner (`state.set_config(key, value)`). Since Redis is flushed after each test, switching back to the original value shouldn't be necessary

Reply:
It's not required. I thought it was, wrote this function, and then realized I didn't need it. I thought I removed this.